style.css

```diff
@@ -230,13 +230,80 @@ button {
 }
 
 .but {
-  /* stylelint-disable-next-line color-hex-length */
-  background-color: #ffffff;
+  background-color: #fff;
   margin: 5px 3px;
   color: #3dc58c;
   border-radius: 5px;
 }
 
+/* add form */
+
+#contact-form {
+  display: flex;
+  flex-direction: column;
+}
+
+#form-element {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  gap: 1rem;
+  margin-top: 2rem;
+  width: 100%;
+}
+
+#form-element li {
+  width: 90%;
+  padding: 0.5rem 0;
+}
+
+input {
+  box-sizing: border-box;
+  height: 3em;
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+textarea {
+  box-sizing: border-box;
+  border: 1px solid #60c095;
+  border-radius: 4px;
+  height: 10em;
+  width: 100%;
+  margin: auto;
+}
+
+input:hover,
+textarea:hover,
```

Comment:
### Styling ✨✨

- [ ] Kindly, in this project, we are supposed to follow the chosen Figma Template design. It would be better to add the background image to replicate the design as much as possible 🙏 

**Your design** | **Figma Design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/228315458-d9391e3c-eb22-445d-8849-9052eb34d9dd.png) | ![image](https://user-images.githubusercontent.com/102757126/228315048-75094868-561a-4d5e-9f32-df34859d0393.png)

- [ ] I think it would be better also to add the [Button interactions](https://www.geeksforgeeks.org/css-buttons/) such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template. 🤩🤩

![image](https://user-images.githubusercontent.com/102757126/228316677-35f337ef-73da-455a-b0fc-b2ada3c5977d.png)